### PR TITLE
:organizations config option

### PR DESCRIPTION
**Pull request by opqdonut**
for #1893

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue

## API
- [x] API is backwards compatible or completely new

## Documentation
- [ ] update changelog if necessary
  - will update once the organizations feature is complete
- [ ] documentation _at least_ for config options (i.e. docs folder)
  - documentation is a task in #1893

## Different installations
- [ ] new configuration options added to rems-deploy repository
  - need to do this for sites that use organizations

## Testing
- [x] valuable features are integration / browser / acceptance tested automatically

## Follow-up
- [x] new tasks are created for pending or remaining tasks

### CODE REVIEW

File: test-config.edn [hunk 1]
```diff
@@ -6,4 +6,6 @@
  :test-database-url "postgresql://localhost/rems_test?user=rems_test"
  :search-index-path "target/search-index-test"
  :authentication :fake-shibboleth
- :enable-pdf-api true}
+ :enable-pdf-api true
+ ;; list all organizations used in test-data
+ :organizations ["" "abc" "nbn" "organization1" "organization2"]}
```
Comment (okahilak) [inline test-config.edn]: Hmm, is "perf" for performance data one of organizations that should be listed here?

File: resources/config-defaults.edn [hunk 2]
```diff
@@ -103,5 +103,8 @@
  ;; :disable-commands [:application.command/assign-external-id :application.command/change-resources]
  :disable-commands []
 
+ ;; List of allowed values for the organization field. Should contain at least one element.
+ :organizations [""]
```
Comment (okahilak) [inline resources/config-defaults.edn]: After thinking a bit more, I'm not sure how "" will work as a default without thinking a few more steps ahead than I'm currently capable of. But I guess this is fine for now and we can change it later on as needed.

File: test-config.edn (hunk 1 above)
Comment (opqdonut) [reply test-config.edn]: good catch, I double-checked the list against the test db

File: resources/config-defaults.edn (hunk 2 above)
Comment (opqdonut) [reply resources/config-defaults.edn]: after discussion, changed it to `["default"]`

**Pull request merged**